src/main/java/com/spotifinder/user/User.java

```diff
@@ -25,10 +28,10 @@ public class User {
     @Column(name = "USERNAME", nullable = false)
     private String username;
 
-    @Column(name = "PASSWORD",nullable = false)
+    @Column(name = "PASSWORD", nullable = false)
     private String password;
 
-    @Column(name = "EMAIL",  nullable = false)
```

Comment:
you can add @Email annotations for data validation

Reply:
Good point. I am gonna add it. Validation will be done on the front-end side. In addition, a few more annotations are still missing here and will be added in the future.